### PR TITLE
Styling Fixes

### PR DESCRIPTION
**Pull request by 007agentHP**
- Created 'Links' and 'More' Screens, applied some basic boiler plate

- Placed Icons into the bottom Navigation Bar

- created and styled header component according to Zeplin

- Minor styling to HomeScreen (Card Icons and Text)

- Added header and created switcher component for 'Busses' page

- wired up Switcher with React-Native-Router-Flux library to allow for switching between 'StopScreen' and 'RouteScreen'.

image

image

image

### CODE REVIEW

File: src/screens/StopScreen.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import Header from '../components/Header';
+import Switcher from '../components/Switcher';
+
+class StopScreen extends React.Component {
+  render() {
+    return (
+      <View style={{ flex: 1, backgroundColor: 'rgb(255, 255, 255)' }}>
+        <Header
+          text={'Bus'}
+        />
```
Comment (ropc) [inline src/screens/StopScreen.js]: (same comment as in `HomeScreen`)

File: src/components/Switcher.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+import { Actions } from 'react-native-router-flux';
+
+//takes 3 props
+//selectedText, object with 2 boolean values selected1 and selected2
+//depending on having the first or second selected
+//text1 for the first portion
+//text2 for the second portion
+class Switcher extends React.Component {
```
Comment (ropc) [inline src/components/Switcher.js]: not sure if this should be extended to work with n-different sections or be specialized to work only with its current usage, which is switching between Stops and Routes in the Bus Screen

File: src/screens/HomeScreen.js [hunk 3]
```diff
@@ -58,22 +67,63 @@ export default class HomeScreen extends React.Component {
       // });
     });
   }
+  //Long Solution that will work for now. Organize better later
+  getDateText() {
```
Comment (007agentHP) [inline src/screens/HomeScreen.js]: Yeah that can change. i'll make the changes accordingly.

File: src/screens/HomeScreen.js [hunk 4]
```diff
@@ -58,22 +67,63 @@ export default class HomeScreen extends React.Component {
...
+    const today = new Date();
+    const weekday = new Array(7);
```
Comment (007agentHP) [inline src/screens/HomeScreen.js]: i'll also make this change

File: src/screens/HomeScreen.js [hunk 5]
```diff
@@ -58,22 +67,63 @@ export default class HomeScreen extends React.Component {
...
+    month[10] = 'November ';
+    month[11] = 'December ';
+
+    const mon = month[today.getMonth()];
+
+    const dayOfMonth = today.getDate().toString();
+
+    const x = mon.concat(dayOfMonth);
+    const y = day.concat(x);
+    return y;
```
Comment (007agentHP) [inline src/screens/HomeScreen.js]: i'll take a closer look into this to make it easier to follow.

File: src/components/Switcher.js (hunk 2 above)
Comment (007agentHP) [inline src/components/Switcher.js]: currently the Switcher is pretty hardcoded but it shouldn't be too difficult to expand it to support n different sections.

Comment (ropc) [inline src/components/Switcher.js]: Yeah. Hardcoding is fine if we intend to use this _just_ for the segmented control in the bus screen. Doing it for n different sections will be trickier but will also be more modular. But either alternative I think is better than the current (which looks like somewhere between the two). I'll let you make that decision.

File: src/screens/StopScreen.js (hunk 1 above)
Comment (007agentHP) [inline src/screens/StopScreen.js]: i'm not sure what you mean by on this.

**Comment by ropc** [comment]
Merging cuz why not

**Pull request merged**